<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="windowTitle"><string>设置</string></property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="group_dataset">
     <property name="title"><string>数据集</string></property>
     <layout class="QGridLayout" name="gridDataset">
      <item row="0" column="0"><widget class="QLabel" name="label_dataset_dir"><property name="text"><string>保存目录</string></property></widget></item>
      <item row="0" column="1"><widget class="QLineEdit" name="dataset_dir_edit"><property name="readOnly"><bool>true</bool></property></widget></item>
      <item row="0" column="2"><widget class="QPushButton" name="choose_dataset_dir_button"><property name="text"><string>选择…</string></property></widget></item>
      <item row="1" column="0"><widget class="QLabel" name="label_last_img_dir"><property name="text"><string>上次图片目录</string></property></widget></item>
      <item row="1" column="1"><widget class="QLineEdit" name="last_img_dir_edit"><property name="readOnly"><bool>true</bool></property></widget></item>
      <item row="1" column="2"><widget class="QPushButton" name="choose_last_img_dir_button"><property name="text"><string>选择…</string></property></widget></item>
      <item row="2" column="0"><widget class="QCheckBox" name="auto_save_checkbox"><property name="text"><string>自动保存</string></property></widget></item>
     </layout>
    </widget>
   </item>

   <item>
    <widget class="QGroupBox" name="group_roi">
     <property name="title"><string>ROI</string></property>
     <layout class="QGridLayout" name="gridRoi">
      <item row="0" column="0"><widget class="QCheckBox" name="fix_roi_checkbox"><property name="text"><string>锁定 ROI 大小</string></property></widget></item>
      <item row="1" column="0"><widget class="QLabel" name="label_roi_w"><property name="text"><string>宽</string></property></widget></item>
      <item row="1" column="1"><widget class="QSpinBox" name="roi_w_spin"><property name="minimum"><number>16</number></property><property name="maximum"><number>4096</number></property><property name="singleStep"><number>16</number></property><property name="value"><number>640</number></property></widget></item>
      <item row="1" column="2"><widget class="QLabel" name="label_roi_h"><property name="text"><string>高</string></property></widget></item>
      <item row="1" column="3"><widget class="QSpinBox" name="roi_h_spin"><property name="minimum"><number>16</number></property><property name="maximum"><number>4096</number></property><property name="singleStep"><number>16</number></property><property name="value"><number>640</number></property></widget></item>
     </layout>
    </widget>
   </item>

   <item><spacer name="spacer_bottom"><property name="orientation"><enum>Qt::Vertical</enum></property><property name="sizeHint" stdset="0"><size><width>20</width><height>40</height></size></property></spacer></item>
   <item>
    <layout class="QHBoxLayout" name="hl_buttons">
     <item><spacer name="spacer_buttons"><property name="orientation"><enum>Qt::Horizontal</enum></property><property name="sizeHint" stdset="0"><size><width>40</width><height>20</height></size></property></spacer></item>
     <item><widget class="QPushButton" name="buttonBox_ok"><property name="text"><string>确定</string></property></widget></item>
     <item><widget class="QPushButton" name="buttonBox_cancel"><property name="text"><string>取消</string></property></widget></item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
